x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -79,6 +93,12 @@ export const searchAfterAndBulkCreate = async ({
     tags,
     throttle,
   });
+  toReturn.lastLookBackDate = new Date(
```

Comment:
I will add a condition on top to not blow your new Date with something weird? Also, I will make sure that you have a number or string before putting in a new Date()